sonar-php-plugin/pom.xml

```diff
@@ -78,8 +78,8 @@
             <configuration>
               <rules>
                 <requireFilesSize>
-                  <minsize>3600000</minsize>
-                  <maxsize>3800000</maxsize>
+                  <minsize>4300000</minsize>
```

Comment:
I created https://jira.sonarsource.com/browse/SONARPHP-736 to minimize the JAR file.